src/aws_s3_bucket_front.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {FormEvent} from 'react';
```

Comment:
thoughts on a different file name? like aws_s3_config_bucket and different class name